src/components/autosuggest/_macro.spec.js

```diff
@@ -5,136 +5,258 @@ import * as cheerio from 'cheerio';
+            });
+
+            test('THEN: it has data-no-results with expected message', () => {
+                expect($('.ons-autosuggest').attr('data-no-results')).toBe('No suggestions found. You can enter your own answer');
+            });
+
+            test('THEN: it has data-results-title with expected title', () => {
+                expect($('.ons-autosuggest').attr('data-results-title')).toBe('Suggestions');
+            });
+
+            test('THEN: it has data-type-more with expected message', () => {
+                expect($('.ons-autosuggest').attr('data-type-more')).toBe('Continue entering to get suggestions');
+            });
+
+            test('THEN: it has the expected id on the results title element', () => {
+                expect($('.ons-autosuggest__results-title').attr('id')).toBe('country-of-birth-suggestions');
+            });
+
+            test('THEN: it renders the results title div with the provided title text', () => {
+                expect($('.ons-autosuggest__results-title').text().trim()).toBe('Suggestions');
+            });
+
+            test('THEN: it has the expected id on the results list element', () => {
+                expect($('.ons-autosuggest__listbox').attr('id')).toBe('country-of-birth-listbox');
+            });
+
+            test('THEN: it renders the results list element with the title attribute set to the provided title text', () => {
+                expect($('.ons-autosuggest__listbox').attr('title')).toBe('Suggestions');
+            });
+
+            test('THEN: it renders instructions with the expected id', () => {
+                expect($('.ons-autosuggest__instructions').attr('id')).toBe('country-of-birth-instructions');
+            });
+
+            test('THEN: it renders the instructions with the provided instructions text', () => {
+                expect($('.ons-autosuggest__instructions').text().trim()).toBe('Use up and down keys to navigate.');
+            });
+
+            test('THEN: the aria-atomic attribute is set to true on the status container', () => {
+                expect($('.ons-autosuggest__status').attr('aria-atomic')).toBe('true');
+            });
+        });
     });
 
-    it('has expected id on container element', () => {
-        const $ = cheerio.load(renderComponent('autosuggest', EXAMPLE_AUTOSUGGEST));
+    describe('GIVEN: Params: minChars', () => {
+        describe('WHEN: minChars is provided', () => {
+            const $ = cheerio.load(
+                renderComponent('autosuggest', {
+                    ...EXAMPLE_AUTOSUGGEST,
+                    minChars: 2,
+                }),
+            );
 
-        expect($('.ons-autosuggest').attr('id')).toBe('country-of-birth-container');
+            test('THEN: it has data-min-chars set to "2"', () => {
+                expect($('.ons-autosuggest').attr('data-min-chars')).toBe('2');
+            });
```

Comment:
We should add a test to check this defaults to 3 when not provided. Could add it in the required section

Reply:
The default is added in the js file, so we should be testing here, same for the threshold test